streams/fr.m3u

```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
 https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="FashionTVCzechSlovak.fr",FashionTV Czech&Slovak (450p) [Not 24/7]
 http://lb.streaming.sk/fashiontv/stream/playlist.m3u8
+#EXTINF:-1 tvg-id="France 2", France 2 (720p)
+http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15226.m3u8
```

Comment:
x-stream codes links are not allowed

Reply:
![image](https://github.com/user-attachments/assets/5496d609-f80d-4a9a-9418-aeed8f00b063)